lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -5848,4 +5905,10 @@ is the only supported options.</xs:documentation>
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="MacroImportType">
+    <xs:restriction base="xs:anyURI">
+      <xs:pattern value="([a-zA-Z0-9_\-/\.])+.xml"/>
```

Comment:
I am not sure if this regex will cover any possible use case or if it will break validation of existing tools.